src/pages/index.tsx

```diff
@@ -100,6 +110,21 @@ export const IndexPageQuery = graphql`
       }
     }
 
+    allContentfulCustomerLogos(
```

Comment:
So did you create a new contentful model? That should be mentioned in the PR
ideally with some notes about how to manage them (or at least a screenshot?)